database/store/cosmos.go

```diff
@@ -38,7 +38,26 @@ func (s *CosmosStore) Has(key []byte) (has bool, err error) {
 	return
 }
 
-// Get retrives service from store. It returns an error if the store does not contains the key.
+// All returns every value from the store.
+func (s *CosmosStore) All() (out [][]byte, err error) {
```

Comment:
This one is rather unused? why we want it here?

Reply:
It was a try and i guess i include the commit by mistake!